mod_reforged/hooks/config/strings.nut

```diff
@@ -1203,8 +1203,8 @@ foreach (vanillaDesc in vanillaDescriptions)
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"As long as your current [Fatigue|Concept.Fatigue] is below " + ::MSU.Text.colorRed("30%") + " of [Maximum Fatigue|Concept.MaximumFatigue] after gear, the [Action Point|Concept.ActionPoints] cost of the first skill used every [turn|Concept.Turn] is " + ::MSU.Text.colorGreen("halved") + ".",
-				"Does not expire when using a skill with no [Action Point|Concept.ActionPoints] and [Fatigue|Concept.Fatigue] cost."
+				"As long as you start your [turn|Concept.Turn] with " + ::MSU.Text.colorRed("30%") + " or less built [Fatigue|Concept.Fatigue], the [Action Point|Concept.ActionPoints] cost of the first skill used is " + ::MSU.Text.colorGreen("halved") + ". Does not expire when using a skill with no [Action Point|Concept.ActionPoints] and [Fatigue|Concept.Fatigue] cost.",
```

Comment:
I don't see the part worthwhile to keep where you can enable this effect when you naturally hit 0 Fatigue. 
Because what is the guy doing in their turn which allows them to naturally reach 0 Fatigue? If they are doing nothing they might aswell use Recover. It's just bloat that may become relevant in 1% of situations.

Suggestion for a cleaner effect text with that change in mind:
```
"The [Action Point|Concept.ActionPoints] cost of the first skill used every [turn|Concept.Turn] is " + ::MSU.Text.colorGreen("halved") + "."
"This effect becomes disabled if you start your turn with 30% or more Fatigue built up, until you use Recover."
```

Reply:
I kinda agree we can simplify it like that. Let's do that and see how it goes.